.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+    pull_request:
+    push:
+        branches: [master]
+
+jobs:
+    cs:
+        name: 'Code Style'
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v2
+            - name: PHP-CS-Fixer
+              uses: docker://oskarstark/php-cs-fixer-ga:2.18.5
```

Comment:
```suggestion
              uses: docker://oskarstark/php-cs-fixer-ga:2.18.6
```

?

Reply:
Oh good, I was about to ask you for a bump. 😅

Reply:
```
Pull down action image 'oskarstark/php-cs-fixer-ga:2.18.6'
  /usr/bin/docker pull oskarstark/php-cs-fixer-ga:2.18.6
  Error response from daemon: manifest for oskarstark/php-cs-fixer-ga:2.18.6 not found: manifest unknown: manifest unknown
  Warning: Docker pull failed with exit code 1, back off 1.367 seconds before retry.
  /usr/bin/docker pull oskarstark/php-cs-fixer-ga:2.18.6
  Error response from daemon: manifest for oskarstark/php-cs-fixer-ga:2.18.6 not found: manifest unknown: manifest unknown
  Warning: Docker pull failed with exit code 1, back off 6.592 seconds before retry.
  /usr/bin/docker pull oskarstark/php-cs-fixer-ga:2.18.6
  Error response from daemon: manifest for oskarstark/php-cs-fixer-ga:2.18.6 not found: manifest unknown: manifest unknown
Error: Docker pull failed with exit code 1
```

Reply:
The tag was not created, can you retry?

Reply:
Looks good now, thanks. 🙂